app/views/shared/_navbar.html.erb

```diff
@@ -19,6 +19,9 @@
           <li class="nav-item">
             <%= link_to "Messages", "#", class: "nav-link" %>
           </li>
+          <li class="nav-item">
+            <%= link_to "My Favorites",favorites_path, class: "nav-link" %>
```

Comment:
put the My Favorites under the avatar dropdown instead. Can replace the another action with my favorites